libs/shared/feature/form-nuv/src/lib/components/date-range-picker/formio/formio-date-range-picker.model.ts

```diff
@@ -97,11 +114,24 @@ function editForm() {
   };
 }
 
-const filteredKeyConfiguration = {
+const startDatePropertyKeyConfiguration = {
+  ...defaultPropertyKeyConfiguration,
+  key: 'components[0].key',
```

Comment:
If you do it like this you don't need an array of components
```suggestion
  key: 'props.startDateKey',
```